.travis.yml

```diff
@@ -62,9 +59,6 @@ matrix:
       dist: trusty
       env: TOXENV='py27-{acme,apache,certbot,dns,nginx}-oldest'
       <<: *not-on-master
-    - python: "3.4"
-      env: TOXENV=py34
```

Comment:
Sorry I didn't notice this in my last review, but can we change this to test Python 3.5? My logic is on PRs it'll keep us testing both the newest and oldest versions of Python 3 we support. Then in our nightly tests, we test on all versions.